src/platform/packages/shared/kbn-unified-data-table/src/components/data_table_additional_display_settings.tsx

```diff
@@ -93,6 +95,43 @@ export const UnifiedDataTableAdditionalDisplaySettings: React.FC<
 
   const settings = [];
 
+  if (onChangeSampleSize) {
+    let step = minRangeSampleSize === RANGE_MIN_SAMPLE_SIZE ? RANGE_STEP_SAMPLE_SIZE : 1;
+
+    if (
+      step > 1 &&
+      ((activeSampleSize && !checkIfValueIsMultipleOfStep(activeSampleSize, step)) ||
+        !checkIfValueIsMultipleOfStep(minRangeSampleSize, step) ||
+        !checkIfValueIsMultipleOfStep(maxAllowedSampleSize, step))
+    ) {
+      step = 1; // Eui is very strict about step, so we need to switch to 1 if the value is not a multiple of the step
+    }
+
+    settings.push(
+      <>
+        <EuiFormRow label={sampleSizeLabel} display="columnCompressed">
+          <EuiRange
+            compressed
+            fullWidth
+            min={minRangeSampleSize}
+            max={maxAllowedSampleSize}
+            step={step}
+            showInput
+            value={activeSampleSize}
+            onChange={onChangeActiveSampleSize}
+            data-test-subj="unifiedDataTableSampleSizeInput"
+            showRange
+          />
+        </EuiFormRow>
+        <EuiHorizontalRule margin="xs" />
```

Comment:
```suggestion
        <EuiHorizontalRule
          margin="xs"
          css={{
            marginInlineStart: `-${euiTheme.size.s}`,
            marginInlineEnd: `-${euiTheme.size.s}`,
            inlineSize: 'unset',
          }}
        />
```
Nit: If we want the divider to match the designs and be full width, we could use negative margins here. Usually I'd recommend getting rid of the popover padding and just adding padding to each `EuiFormRow`, but we don't have that control for the data grid popover.

Reply:
Oh wow, what a good eye! I didn't even notice it, it's so subtle!